packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -41,7 +41,7 @@ function findQueryTypeOrThrow(queryName: string, schema: IntrospectionQuery) {
     return ret;
 }
 
-export type Prop = { type: string; optional: boolean; name: string };
+export type Prop = { type: string; optional: boolean; name: string; destructionAlias?: string };
```

Comment:
destructionAlias is not used, what are the plans with it?

Reply:
I'm not using it anymore. I'm removing it.